### ISSUE TITLE
Sidemenu JS should not add aria-expanded="false" to .mzp-c-sidemenu-main

### ISSUE THREAD

**Issue by alexgibson** [mozilla/protocol#999]
## Description

Similar to https://github.com/mozilla/protocol/issues/847, the `aria-expanded` attribute is being applied incorrectly to the side menu component.

This role should be declared on a controlling element like a button or link and must be used with aria-controls. Also, on desktop this content is expanded by default so it should not be false.
